Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 
 gemspec
+
+gem 'byebug'
```

Comment:
I suggest you can remove byebug-related codes, for 2 reasons:
1. not all developers use `byebug` to debug
2. the dependencies should be added in `.gemspec`, bundler here is for the sandbox environment while developing this gem.

Reply:
@kevinluo201 Hi, that's actually my goal here. I use byebug to help me debug when I work on this gem. I don't want to impose byebug to developers who use that gem.